ERCS/erc-7565.md

```diff
@@ -0,0 +1,327 @@
+---
+eip: 7565
+title: Perpetual Contract NFTs as Collateral for Liquidity Provision
+description: A standard for representing locked financial assets as Non-Fungible Tokens (NFTs), enabling these NFTs to be used as collateral for borrowing funds in Decentralized Finance (DeFi). This ERC also integrates the concept of rentable NFTs (via ERC-4907) to facilitate liquidity provision.
+author: Hyoungsung Kim (@HyoungsungKim) <hyougnsung@keti.re.kr>, Yong-Suk Park <yspark@keti.re.kr>, Hyun-Sik Kim <hskim@keti.re.kr>
+discussions-to: https://ethereum-magicians.org/t/erc-7565-proposal-perpetual-contract-nft-for-defi-composability/16790
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-11-27
+requires: 721, 4907
+---
+
+## Abstract
+
+This ERC proposes a mechanism where a person (referred to as the "Asset Owner") can collateralize NFTs that represent locked deposits or assets, to borrow funds against them. These NFTs represent the right to claim the underlying assets, along with any accrued benefits, after a predefined maturity period. For an academic article, please visit [IEEE Xplore](https://ieeexplore.ieee.org/document/9967987/).
```

Comment:
Please cite this external resource using the [format defined in EIP-1](https://eips.ethereum.org/EIPS/eip-1#digital-object-identifier-system). It might look something like this:

````suggestion
This ERC proposes a mechanism where a person (referred to as the "Asset Owner") can collateralize NFTs that represent locked deposits or assets, to borrow funds against them. These NFTs represent the right to claim the underlying assets, along with any accrued benefits, after a predefined maturity period. [^1]

[^1]:
    ```csl-json
    {
        "container-title": "IEEE Access",
        "author": [
            {
                "given": "Hyoungsung",
                "family": "Kim"
            },
            {
                "given": "Hyun-Sik",
                "family": "Kim"
            },
            {
                "given": "Yong-Suk",
                "family": "Park"
            }
        ],
        "DOI": "10.1109/ACCESS.2022.3225884",
        "URL": "https://ieeexplore.ieee.org/document/9967987",
        "type": "article-journal",
        "id": 9967987,
        "citation-label": "9967987",
        "issue": null,
        "issued": {
            "date-parts": [
                [
                    2022
                ]
            ]
        },
        "keyword": "Contracts;Nonfungible tokens;Cryptocurrency;Finance;Smart contracts;Blockchains;Financial services;Automated market maker (AMM);blockchain;decentralized exchange (DEX);decentralized finance (DeFi);Ethereum;liquidity pool (LP);non-fungible token (NFT);uniswap",
        "page": "126802-126814",
        "title": "Perpetual Contract NFT as Collateral for DeFi Composability",
        "volume": 10
    }
    ```
````